README.md

```diff
@@ -79,6 +79,65 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
```

Comment:
I think it makes sense given how this is flowing to move the Build from Source to a BUILD.md like ponyc has.

It's existence between the expected installation method and these instructions is problematic. It breaks up the flow that you'd expect most folks to follow. Upon seeing it, you were definitely right to suggest that change. That could be another change in which case I think that `Getting started using Corral` should come before the build from source.

Either change is fine for now. Your call.